.template/addons/nginx/config/nginx/app.conf.template.tt

```diff
@@ -0,0 +1,113 @@
+worker_processes auto;  ## Default: 1
+
+events {
+  worker_connections 1024;  ## Default: 1024
+}
+
+http {
+  include /etc/nginx/mime.types;
+
+  gzip on;
+  gzip_comp_level 5;
+  gzip_min_length 256;
+  gzip_proxied any;
+  gzip_vary on;
+  gzip_types
+    application/atom+xml
+    application/geo+json
+    application/javascript
+    application/json
+    application/ld+json
+    application/manifest+json
+    application/rdf+xml
+    application/rss+xml
+    application/vnd.ms-fontobject
+    application/wasm
+    application/x-web-app-manifest+json
+    application/xhtml+xml
+    application/xml
+    font/otf
+    image/bmp
+    image/svg+xml
+    text/cache-manifest
+    text/calendar
+    text/css
+    text/javascript
+    text/markdown
+    text/plain
+    text/vcard
+    text/vnd.rim.location.xloc
+    text/vtt
+    text/x-component
+    text/x-cross-domain-policy;
+
+  upstream app {
+    server localhost:3000;
+  }
+
+  server {
+    server_name _;
+
+    root /<%= APP_NAME %>/public;
```

Comment:
is the `<%= APP_NAME %>` good example for Nginx config?

Normally I use this one:

```suggestion
    root /var/www/html/<%= APP_NAME %>/public;
```

Or

```suggestion
    root /srv/apps/<%= APP_NAME %>/public;
```

Reply:
normally we deploy the Rails app to the root folder @byhbt it's different with PHP as the app locates in `var/www/html`.

Reply:
ok. LGTM